src/components/Tooltip/Tooltip.jsx

```diff
@@ -90,11 +90,12 @@ class Tooltip extends React.Component {
           </Styles.HotkeyWrapper>
         </Styles.Label>
       )}
+      {customHTML && customHTML}
```

Comment:
I don't know if you need to use `customHTML && customHTML` and not just `customHTML` because if the content is null-ish it will just not render. Might be worth checking that 🙂